src/applications/search/containers/SearchApp.jsx

```diff
@@ -125,7 +125,7 @@ class SearchApp extends React.Component {
     });
   };
 
-  onSearchResultClick = (bestBet, strippedTitle, index) => () => {
+  onSearchResultClick = (bestBet, strippedTitle, index, url) => () => {
```

Comment:
GM... can we pass an object instead to reduce the number of parameters passed to this function? I was thinking something like:

```javascript 
searchConfig = {
isBestBet,
strippedTitle
index
result.url
}

Reply:
Thanks for the suggestion! I have updated it to help limit the sprawl of parameters!